website/content/prototyping/_index.markdown

```diff
@@ -1,195 +1,17 @@
 ---
 layout: default
-title: Non-Go Prototyping
+title: Package development
```

Comment:
I kept the `/prototyping/` URI, since the current `Non-Go Prototyping` page actually starts with go sections:
![2022-04-10-134058](https://user-images.githubusercontent.com/3864879/162616348-4ce63857-db44-4c43-940d-9fce4c21f6a9.png)

The `Go software not written for gokrazy:` remains available at this URI (only the non-go stuff changed location).

I am not found of using `/development/` for this. I think it would be better to document how to contribute to gokrazy under this URI (supervisor, packer...).

What do you think?

Reply:
I think `/contributing/` would be better for documentation about contributing :) In general, the URL should match the title.

I understand the split between Go and non-Go software, but I still think the `/prototyping/` URL should lead to the non-go prototyping instructions — I have linked to it in various contexts, so it would not be great if we moved the content away.